web/app/login/auth/redirectOnAuth.tsx

```diff
@@ -10,12 +10,12 @@ interface RedirectOnAuthProps {
 export default function RedirectOnAuth({ page }: RedirectOnAuthProps) {
   const { user } = useUserInfo();
   const router = useRouter();
-  useEffect(() => {
-    if (localStorage.getItem("isLoggedIn") === "true" && !user) {
-      return;
-    }
+  useEffect(() => {    if (localStorage.getItem("isLoggedIn") === "true" && !user) {
```

Comment:
Is this formatted with prettier?

Reply:
This line no but I reformatted this with prettier in a new commit